etc/.hadolint.yml

```diff
@@ -3,3 +3,4 @@ ignored:
   # Issue when using pipenv, which is our standard. Able to workaround with an
   # environment variable of PIP_NO_CACHE_DIR=1
   - DL3042
+  - DL3028
```

Comment:
Please provide a justification, similar to the prior comments in the file

Reply:
@JonZeolla I noticed that a lot of packages are being stripped from super-linter for goat. I had to add back go even though it should be in super-linter [by default](https://github.com/github/super-linter/blob/ae3c77f45b03c6e02021b04fd62abf51046434a0/Dockerfile#L63). Checking here, ruby support should [already exist](https://github.com/github/super-linter/blob/ae3c77f45b03c6e02021b04fd62abf51046434a0/Dockerfile#L86) as well. I think a lot of this is just extra work from super-linter being super-linter.

Reply:
Yeah we have seen that for a while.  I still don't know why or how that's happening.

@digeseiso still waiting for a justification here (in the file; see above comment)

Reply:
@JonZeolla added justification